src/partials/_sell.hbs

```diff
@@ -1,13 +1,13 @@
-<section class="sell">
-  <h2>Sell A Modern Eden</h2>
-  <p class="type--p__large">Own a boutique or store and looking to sell products from A Modern Eden? Let us know and we will provide you with everything you need to know about becoming an exclusive vendor.</p>
-  <p class="type--p__large">Have general quesitons about the A Modern Eden products, feel free to drop us a line.</p>
+<section class="sell clearfix">
+  <h2 class="sell--header">Sell A Modern Eden</h2>
+  <p class="sell--type">Own a boutique or store and looking to sell products from A Modern Eden? Let us know and we will provide you with everything you need to know about becoming an exclusive vendor.</p>
+  <p class="sell--type">Have general quesitons about the A Modern Eden products, feel free to drop us a line.</p>
   <form class="sell--form" action="index.html" method="post">
-    <label for="name">name</label>
-    <input type="text" name="name" value="" placeholder="Your name">
-    <label for="email">email</label>
-    <input type="email" name="email" value="" placeholder="Your email">
-    <input type="textarea" name="message" placeholder="I would like to learn more about how my business can sell A Modern Eden products.">
-    <input type="submit" value="submit">
+    <label class="sell--label" for="name">name</label>
+    <input class="sell--input" type="text" name="name" value="" placeholder="Your name">
```

Comment:
- I think the `for` attributes on these labels should refer to the `id`s of their respective inputs, but the inputs don't have `id`s. 
- If the `value` attr is empty, why not remove it?